ccd-definition/CaseEvent/User/UserEvents-SDO-nonprod.json

```diff
@@ -16,8 +16,7 @@
     "EndButtonLabel": "Submit",
     "RetriesTimeoutAboutToStartEvent": 0,
     "RetriesTimeoutURLAboutToSubmitEvent": 0,
-    "Publish": "Y",
```

Comment:
This needs reverted before merge, changed for testing

Reply:
Done. Resolving this conversation.